server/master/src/main/java/org/apache/accumulo/master/replication/DistributedWorkQueueWorkAssigner.java

```diff
@@ -67,45 +67,18 @@ protected boolean isWorkRequired(Status status) {
   protected int maxQueueSize;
   protected ZooCache zooCache;
 
-  /*
-   * Getters/setters for testing purposes
-   */
-  protected AccumuloClient getClient() {
-    return client;
-  }
-
   protected void setClient(AccumuloClient client) {
```

Comment:
With the other API changes in this class, I'm wondering if it makes sense to make client a constructor param.

Reply:
The two child classes have constructors that take `(AccumuloConfiguration conf, AccumuloClient client)`.  This method is only used for tests so we probably just need to rewrite the tests to use the children.  I can open a follow on ticket.

Reply:
Opened up #876